crime-applications-adaptor/application/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/BenefitsMapper.java

```diff
@@ -5,29 +5,33 @@
 import java.util.Objects;
 import org.apache.commons.lang3.StringUtils;
 import uk.gov.justice.laa.crime.applications.adaptor.enums.BenefitDetails;
-import uk.gov.justice.laa.crime.applications.adaptor.factory.PoundSterling;
-import uk.gov.justice.laa.crime.applications.adaptor.util.FrequencyMapper;
+import uk.gov.justice.laa.crime.applications.adaptor.util.AssessmentDetailMapperUtil;
 import uk.gov.justice.laa.crime.model.common.crimeapplication.common.AssessmentDetail;
 import uk.gov.justice.laa.crime.model.common.criminalapplicationsdatastore.general.IncomeBenefit;
 
 public class BenefitsMapper {
 
   private static final String UNIVERSAL_CREDIT = "Universal Credit";
   private static final String JSA = "Contribution-based Jobseeker Allowance";
+  protected static final String DEFAULT_OWNERSHIP_TYPE = "applicant";
 
   public List<AssessmentDetail> mapBenefits(List<IncomeBenefit> benefits) {
     List<AssessmentDetail> assessmentDetails = new ArrayList<>();
 
     if (Objects.nonNull(benefits)) {
```

Comment:
I understand this is outside of your immediate changes, I would consider returning early if `benefits` are `null` (or empty?) to reduce the nesting.
e.g.
```
    if (CollectionUtils.isEmpty(benefits)) {
      return assessmentDetails;
    }
```

Reply:
I think we are OK with the existing code